### ISSUE TITLE
Update colours on plotting functions

### ISSUE THREAD

**Issue by katielukow-AE** [About-Energy-OpenSource/AEPyBaMM#6]
The color scheme on the plotting functions should be updated for ease of understanding and accessibility

**Comment by darryl-ad** [comment]
See also:
- DFN Simscape - https://github.com/AboutEnergy/Simscape-DFN-Model/issues/82
- ECM - https://github.com/AboutEnergy/ECM/issues/133
